client/state/login/magic-login/actions.js

```diff
@@ -92,6 +89,20 @@ export const fetchMagicLoginRequestEmail = ( email, redirectTo ) => dispatch =>
 		} );
 };
 
+async function postMagicLoginRequest( url, bodyObj ) {
+	const response = await fetch( url, {
+		method: 'POST',
+		credentials: 'include',
+		headers: { Accept: 'application/json', 'Content-Type': 'application/x-www-form-urlencoded' },
+		body: stringify( bodyObj ),
+	} );
+
+	if ( response.ok ) {
+		return await response.json();
+	}
+	throw new HTTPError( response.status, await response.body );
```

Comment:
The first constructor param for `HTTPError` is `response`, not `status`.

Reply:
Great catch! 👍